x.toml

```diff
@@ -3,7 +3,6 @@ version = "0.8.2"
 
 [system-tests]
 jsonrpc-integration-tests = { path = "json-rpc/integration-tests" }
-shuffle-integration-tests = { path = "shuffle/integration-tests" }
```

Comment:
To reiterate, let's not tinker with the forge test setup and only focus on cli tests this PR. It'll make the diff much smaller and easier to review.